lang/en.json

```
@@ -312,6 +324,7 @@
     "RollTypeSpell": "Spell Roll",
     "RollWithAnEdge": "with an Edge",
     "RollWithASnag": "with a Snag",
+    "SelectionLimit": "Number of Times Selected",
```

```suggestion
    "PowerSelectionLimit": "Selection Limit",
```
To me "Number of Times Selected" sounds like how many times this item has be selected already, not the max.

---

Yea that one was weirdly worded.